build-tools/automation/azure-pipelines.yaml

```diff
@@ -405,7 +405,7 @@ stages:
       parameters:
         configuration: $(ApkTestConfiguration)
 
-    - script: 'brew uninstall openssl@1.0.2t && rm -rf /usr/local/etc/openssl && rm -rf /usr/local/etc/openssl@1.1'
+    - script: 'brew uninstall -f openssl@1.0.2t && rm -rf /usr/local/etc/openssl && rm -rf /usr/local/etc/openssl@1.1'
```

Comment:
I believe the issue that we needed this workaround for was fixed last week, I think we can remove this line entirely now instead.